{{ cookiecutter.project_name }}/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,63 @@
+#### Description
+
+<!-- Please include a summary of the change and the relevant issue(s) it
+resolves, if any (otherwise delete that line), e.g., `Fixes #123`. If the PR
+addresses more than one issue, please add multiple lines, each starting with
+'Fixes #'. Please stick to that syntax precisely, including whitespaces,
+otherwise the issue(s) may not be linked to the PR.
+
+In the summary, list any dependencies that are required for this change.
+Please use bullet points for the description. Please also briefly describe
+the relevant motivation and context briefly. For very trivial changes that are
+duly explained by the PR title, a description can be omitted. -->
+
+- Fixes #(issue number)
+
```

Comment:
Add an example, e.g.,:

```
<!-- Example:

Fixes #2
Fixes #3

- Address bug X by Y
- Add support for feature X through Y

-->
```

Strangely, the majority of people get this wrong and PRs are never linked to the corresponding issues.

Reply:
```
- Address bug X by Y
- Add support for feature X through Y
```
Is this necessary, if we have linked, it do we need to also specify the bug when it has already been detailed in the linked issue?

Reply:
Yes, though Sourcery nowadays will do it for us.

It's necessary because a PR description is a description of the proposed changes to address a bug or a feature etc - an issue is a description of a bug or a feature itself. It's a difference. By reading the linked issue, you will not necessarily have an idea of what exactly has changed in the codebase.

Reply:
yeah ig so :+1: